{
  "uid" : "ba40dacb3fb32141",
  "name" : "test_1",
  "fullName" : "Test.test_search.Test_Base#test_1",
  "historyId" : "fe554796863e4952e0591ffbf8edb4fc",
  "time" : {
    "start" : 1572390915191,
    "stop" : 1572390916431,
    "duration" : 1240
  },
  "status" : "passed",
  "flaky" : true,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "Test"
  }, {
    "name" : "suite",
    "value" : "test_search"
  }, {
    "name" : "subSuite",
    "value" : "Test_Base"
  }, {
    "name" : "host",
    "value" : "DESKTOP-E3BR8K0"
  }, {
    "name" : "thread",
    "value" : "19220-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "Test.test_search"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "ebd56493ac71cccb",
      "status" : "broken",
      "statusDetails" : "urllib.error.URLError: <urlopen error [WinError 10061] 由于目标计算机积极拒绝，无法连接。>",
      "time" : {
        "start" : 1572390813606,
        "stop" : 1572390813606,
        "duration" : 0
      }
    }, {
      "uid" : "376e4cbbebf36e1e",
      "status" : "broken",
      "statusDetails" : "urllib.error.URLError: <urlopen error [WinError 10061] 由于目标计算机积极拒绝，无法连接。>",
      "time" : {
        "start" : 1572390772266,
        "stop" : 1572390772266,
        "duration" : 0
      }
    } ],
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "ba40dacb3fb32141.json",
  "parameterValues" : [ ]
}